src/Api/Response/ListAddressResponse.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Api\Response;
+
+/**
+ * Response for address list requests
+ */
+class ListAddressResponse extends AddressResponse
+{
+    /**
+     * @return array
+     */
+    public function getResults(): array
+    {
+        return $this->data['results'] ?? [];
+    }
+
+    /**
+     * @return void
+     */
+    protected function parseResponseBody(): void
+    {
+        parent::parseResponseBody();
+        $this->data = ['results' => $this->data['results'] ?? []];
```

Comment:
Dit is niet blocking voor mijn goedkeuring, maar waarom schrijven we niet iets meer wat we bedoelen en voorkomen daarmee een extra assignment?
``` 
if (!isset($this->data['results']) { $this-data['results'] = []; }
```